src/azure-cli/azure/cli/command_modules/rdbms/commands.py

```diff
@@ -420,3 +434,21 @@ def load_command_table(self, _):
         g.custom_command('delete', 'server_key_delete', confirmation=True)
         g.custom_show_command('show', 'server_key_get')
         g.command('list', 'list')
+
+    with self.command_group('mysql server ad-admin',
+                            mysql_adadmin_sdk,
+                            client_factory=cf_mysql_server_ad_administrators_operations,
+                            is_preview=True) as g:
+        g.custom_command('create', 'server_ad_admin_set')
+        g.command('list', 'list')
+        g.command('delete', 'delete')
+        g.command('show', 'get')
```

Comment:
We don't support update the ad-admin info for a server, right?

Reply:
Yes